src/components/study-manager.js

```diff
@@ -474,23 +486,24 @@ const StudyManager = ({ onClick }) => {
                             />
                         </Grid>
                     ))}
-                {studies.map((study) => (
-                    <Grid
-                        item
-                        xs={12}
-                        sm={6}
-                        md={3}
-                        key={study.userId + '/' + study.studyName}
-                    >
-                        <StudyCard
-                            studyCreationLoader={false}
-                            study={study}
-                            onClick={() =>
-                                onClick(study.studyName, study.userId)
-                            }
-                        />
-                    </Grid>
-                ))}
+                {studies instanceof Array &&
```

Comment:
In which case studies is not an array even if empty?

Reply:
I will do that

Reply:
Done